view/src/components/Difference.jsx

```diff
@@ -1,15 +1,15 @@
 // Component for visualisation of found difference
 // Author: Lukas Petr
 
-import { useState } from 'react';
+import { useMemo, useState } from 'react';
 import { PropTypes } from 'prop-types';
 
 import Container from 'react-bootstrap/Container';
 import Col from 'react-bootstrap/Col';
 import Row from 'react-bootstrap/Row';
 import Alert from 'react-bootstrap/Alert';
 
-import Callstack from './Callstack';
+import Callstack, { CallSide } from './Callstack';
```

Comment:
Relevant comment from #314:
> Shouldn't be?
> ```suggestion
> import { Callstack, CallSide } from './Callstack';
>   ```

I do not think so, the component `CallStack` is exported as default export, therefore it is outside of the braces, but `CallSide` is exported as named export, therefore it is in the braces ([import documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#syntax)). I could change the way how it is exported, but I exported the `CallStack` as default because it is the main thing from the file, furthermore I think I saw somewhere this approach - eg. [here](https://create-react-app.dev/docs/importing-a-component/), where in old (with class approach) React you did something like  `import React, { Component } from 'react';`)

Reply:
I think that it is all right, but I do not have much experience with clean javascript code. But your point makes sense, especially if you are using it elsewhere in the project.